documentation/4.0/content/managing-domains/domain-lifecycle/liveness-readiness-probe-customization.md

```diff
@@ -78,6 +78,26 @@ The following operator-populated environment variables are available for use in
 
 * A custom liveness probe must not fail (exit non-zero) when the WebLogic Server instance itself is unavailable. This could be the case when the WebLogic Server instance is booting or about to boot.
 
+### Automatic Restart of Failed Server Instances by Node Manager
```

Comment:
Automatic Restart of Failed Server Instances by Node Manager -> Automatic restart of failed server instances by Node Manager

Reply:
Changed in [a1b5893](https://github.com/oracle/weblogic-kubernetes-operator/pull/3706/commits/a1b5893699226c37d6167b27dec280dbe5aa240a). Thanks.